src/coreclr/tools/aot/Mono.Linker.Tests/Mono.Linker.Tests.csproj

```
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Platforms>x64;x86</Platforms>
+    <PlatformTarget>AnyCPU</PlatformTarget>
+
+    <RuntimeIdentifiers>linux-x64;win-x64;osx-x64</RuntimeIdentifiers>
+    <Configurations>Debug;Release;Checked</Configurations>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="FluentAssertions" Version="6.0.0-alpha0002" />
+
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="$(MicrosoftCodeAnalysisCSharpVersion)" />
+
+    <PackageReference Include="Mono.Cecil" Version="0.11.4" />
+    <PackageReference Include="System.Reflection.MetadataLoadContext" Version="5.0.1" />
+    <ProjectReference Include="../Mono.Linker.Tests.Cases/Mono.Linker.Tests.Cases.csproj" />
+    <ProjectReference Include="../Mono.Linker.Tests.Cases.Expectations/Mono.Linker.Tests.Cases.Expectations.csproj" />
+    <ProjectReference Include="../ILCompiler/ILCompiler.csproj" />
```

It's odd to add an EXE as a ProjectReference:

```suggestion
    <ProjectReference Include="../ILCompiler.Compiler/ILCompiler.Compiler.csproj" />
    <ProjectReference Include="../ILCompiler.RyuJit/ILCompiler.RyuJit.csproj" />

```

---

Sorry - that's a leftover from my trying things out... and I forgot about it. We should definitely change this - we don't need/want the exe for anything.